CMakeLists.txt

```diff
@@ -378,7 +378,6 @@ IF(WANT_JACK)
 			SET(LMMS_HAVE_WEAKJACK TRUE)
 			SET(WEAKJACK_INCLUDE_DIRS ${CMAKE_SOURCE_DIR}/src/3rdparty/weakjack/weakjack)
 			SET(STATUS_JACK "OK (weak linking enabled)")
-			SET(JACK_INCLUDE_DIRS "")
```

Comment:
If I recall correctly, this was a throwback to our unorthodox method of including variables that weren't ever set inside [`src/`](https://github.com/LMMS/lmms/blob/e614711d29d5ee5a958da55f4bb3716d942cd5c6/src/CMakeLists.txt#L54-L55) to avoid cmake warnings or failures.  This is undoubtably the wrong approach so as long as `jack.pc` sets this correctly (and from my experience, `pkg-config` is very consistent at this).  Perhaps it should be set to blank if it WASN'T found.

Reply:
@devnexen thanks for the attention to detail here. 👍 Travis-CI was hung, kick-started it and it's good.  Merging.  If this causes issues in the future it's easy enough to put an addition check in place.